### DISCUSSION TITLE
how to accept the invitation by classroom bot

### DISCUSSION THREAD

**Discussion by Debangshu97** [open-sauced-craftwork/browser-extension#17] [Q&A] [answered]
You can [accept or decline] this invitation. You can also visit [@github-classroom[bot]] to learn a bit more about them.

This invitation will expire in 7 days.

How to actually accept or decline the invitation. I am unable to find any button as such

**Comment by doaortu** [comment]
When you can access the repo open_sauced_craftwork_browser_extension-<yourgithubusername> , you've already accept it, i guess.

**Comment by bdougie** [comment, marked as answer]
No automation is present in this assignment. 

https://github.com/open-sauced-craftwork/browser-extension/issues/9

**Reply by thekingslee** [reply]
How do you mean? I am a lil confused

**Reply by bdougie** [reply]
I did not configure the bot, so whether the invite is accepted or not does not matter.

**Reply by thekingslee** [reply]
Okay
